### DISCUSSION TITLE
v1.9.3 draft

### DISCUSSION THREAD

**Discussion by GeckoEidechse** [R2Northstar/Northstar#321] [Draft patch notes]
**Features:**
Allow custom maps to set Fastball panels ([#475](https://github.com/R2Northstar/NorthstarMods/pull/475))
Support to input other languages in the search box in serverbrowser ([#477](https://github.com/R2Northstar/NorthstarMods/pull/477))

**Fixes:**
Removal of live fire map from titan tag ([#453](https://github.com/R2Northstar/NorthstarMods/pull/453))
Fixed RespawnAsTitan not triggering FirstToFall ([#427](https://github.com/R2Northstar/NorthstarMods/pull/427))
Fixed killer outline missing in kill replay ([#470](https://github.com/R2Northstar/NorthstarMods/pull/470))
Fixed turret not showing laser indicator ([#467](https://github.com/R2Northstar/NorthstarMods/pull/467))
Added IsValid Check to DelayedGamemodeAnnouncement ([#471](https://github.com/R2Northstar/NorthstarMods/pull/471))
Fixed race condition crash when boarding dropship as it explodes ([#438](https://github.com/R2Northstar/NorthstarMods/pull/438))

**Contributors:**
@BobTheBob9 @catornot @DBmaoha @H0L0theBard @x3Karma @zxcPandora

**Comment by uniboi** [comment]
Isn't serverside rui a new feature as well?

**Reply by GeckoEidechse** [reply]
Serverside RUI is a bigger feature and as this is just a patch release, it's not included here. Serverside RUI will be in 1.10
